modules/console/pages/config/security/authentication.adoc

```diff
@@ -1,61 +1,367 @@
+      N --> U{User impersonation enabled?}
+      U -- Yes --> V[Forward JWT credentials\nheader-based authentication]
+      U -- No --> W[Use static credentials\n+ enforce console RBAC]
+    end
+....
+
+=== OIDC authentication
+
+When using OIDC, the user is redirected to an external identity provider, such as Google, for authentication. Upon successful login, the provider issues an ID token which is then captured and encapsulated within a JWT. This token is used in the following ways:
+
+* **Kafka authentication:**
+  The ID token is mapped to SASL-OAUTHBEARER, allowing Kafka to authenticate requests under the user's identity.
+
+* **HTTP API authentication:**
+  The ID token is used in header-based authentication (`Authorization: Bearer <id-token>`) in the Schema Registry and Admin API.
 
-* xref:./azure-ad.adoc[Microsoft Entra ID]
-* xref:./google.adoc[Google]
-* xref:./github.adoc[GitHub]
-* xref:./keycloak.adoc[Keycloak]
-* xref:./okta.adoc[Okta]
-* xref:./generic-oidc.adoc[Generic OIDC]
+=== Basic authentication
 
-You can use one or more login providers at the same time. To enable SSO authentication,
-you must create an OAuth application for your organization first. Refer to the respective
-documentation page for guidance to set up your desired identity provider in Redpanda Console.
-Afterwards, you can configure your identity provider in Redpanda Console by providing the
-clientId and clientSecret in the configuration block for your provider. The configuration
-to add Google login support looks like this:
+For basic authentication, the user submits a username and password directly to Redpanda Console. These credentials are validated and encapsulated within a JWT, which is then used as follows:
+
+* **Kafka authentication:**
+  The provided credentials are mapped to SASL-SCRAM, ensuring that Kafka operations execute with the correct user identity.
+
+* **HTTP API authentication:**
+  The JWT supports HTTP Basic authentication for other APIs.
+
+=== User impersonation
+
+User impersonation determines how the user's identity (as captured in the JWT) is propagated to upstream APIs. This can be configured on a per-API basis:
+
+* **Impersonated users:**
+  When impersonation is enabled for an API, Redpanda Console uses the logged-in user's credentials from the JWT to authenticate API requests. This ensures that actions are executed under the user's actual identity and that authorization is deferred to Redpanda. In this mode, Redpanda Console's role-based access control (RBAC) is bypassed for the impersonated API calls.
+
+* **Non-impersonated users (static credentials):**
+  When impersonation is disabled, Redpanda Console does not forward the user's credentials. Instead, static credentials must be provided in the configuration, and Redpanda Console role bindings determine the user's access to Redpanda Console features. In this scenario, the Console's RBAC rules (admin, editor, viewer) define what the user can see and do.
```

Comment:
In this block I'm wondering whether it's actually clear that that this can be done per API (Kafka, SR, AdminAPI), maybe a small example would be helpful to show that it can be done per API? I'm unsure, was just my initial thought that the `per-API basis` mention is very critical.